content/cloud/cloud-command-reference.md

```diff
@@ -1608,6 +1887,48 @@ OPTIONS:
 
 {{ blockEnd }}
 
+{{ startTab "v0.6.0"}}
```

Comment:
I think there should be a `logout` command between `login` and `logs` (unexciting though it is).

Reply:
Thanks @itowlson - good spotting